hw/ip/pattgen/dv/pattgen_sim_cfg.hjson

```diff
@@ -12,7 +12,7 @@
   tb: tb
 
   // Simulator used to sign off this block
-  tool: vcs
+  tool: xcelium,
```

Comment:
the tailing `,` isn't needed. maybe this causes the CI failure

Reply:
@weicaiyang thanks, you same my time to debug CI error